modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/InitJoinDump.pm

```diff
@@ -58,19 +58,19 @@ sub run {
       closedir($dh);
       foreach my $file (@dir_content) {
         next if ($file =~ m/^\./);
-        if ($file =~ m/\.$file_type\.gz/) {
+        # homo_sapiens_structural_variations-chr10.gvf
+        # homo_sapiens_structural_variations-chr10.vcf.gz 
+        if ($file =~ m/\.$file_type\.gz$|\.$file_type/) {
```

Comment:
My regular expressions are a little rusty, but wont anything that matches `/\.$file_type/` also match `/\.$file_type.gz/`? I suggest` /\.($file_type\.gz|$file_type)$/` (although I haven't tested it)